JustSaying/MessageResponseLogger.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Threading.Tasks;
+using JustSaying.AwsTools.MessageHandling;
+
+namespace JustSaying
+{
+    public class MessageResponseLogger
+    {
+        public MessageResponseLogger()
+        {
+            ResponseLogger = _ => { };
+            ResponseLoggerAsync = _ => Task.FromResult(0);
```

Comment:
Can you use `Task.CompletedTask` here too, or are we targeting too low a TFM for it?

Reply:
Too lower framework.